browser-facade/aspnetapp/Pages/Friends.cshtml

```diff
@@ -9,7 +9,9 @@
     @foreach (var conversation in Model.Conversations)
     {
         <li>
-            Conversation between @conversation.Sender and @conversation.Receiver
+            <a asp-page="/Messages" asp-route-receiver="@conversation.Receiver">
```

Comment:
AFAIK conversation Sender and Receiver are determined by who messaged first. Thus, if `Kuba` messages `brud` first, the Receiver for both of them will be `brud`, no matter who is logged in.

To be fixed in #56